### DISCUSSION TITLE
从节点开始往空白处画边，为什么会出现无法画出直线？

### DISCUSSION THREAD

**Discussion by Lyong0216** [antvis/X6#2645] [Q&A]
正常从节点开始往空白处画边，边的结尾处一直无法呈现直线的状态，如下图：
![image](https://user-images.githubusercontent.com/72903238/188821895-c88f6d3c-9cd1-4867-b059-c3a436783bc7.png)
求解决方法，谢谢！

**Comment by zmcode** [comment]
展示你的代码示例
